core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sort/SortOpDesc.scala

```diff
@@ -0,0 +1,51 @@
+package edu.uci.ics.texera.workflow.operators.sort
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.Schema
+
+class SortOpDesc extends PythonOperatorDescriptor {
+  @JsonProperty(required = true)
+  @JsonPropertyDescription("column to perform sorting on")
+  var attributes: List[SortCriteriaUnit] = _
+
+  override def generatePythonCode(): String = {
+    val criteriaString: String = "[" + attributes
+      .map { criteria =>
+        s"""("${criteria.attributeName}", ${criteria.sortPreference match {
+          case SortPreference.ASC  => "\"ASC\""
+          case SortPreference.DESC => "\"DESC\""
+        }})"""
```

Comment:
better not to pass string values and do pattern matching later. try to pass explicit boolean values instead?

Reply:
OK